static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -236,6 +250,19 @@ define([
         postDisplay = function () {
             mediator.on('window:resize', windowResize);
         },
+        showAdsFreeSurvey = function (size, $adSlot) {
+            fastdom.write(function () {
+                var crossIcon = svgs('crossIcon'),
+                    dataAttr = isAdfreeSurvey('variant') ? 'hide ads' : 'hide ads simple',
+                    $adSlotRemove = $(document.createElement('div')).addClass('ad-slot--remove').attr('data-link-name', dataAttr)
```

Comment:
This is 404 survey, so I assume that in the future there will be a real option to remove those ads. I get your idea but don't think that it is necessary to change the css class now.